kubernetes/services/nginx-storage.yml.j2

```diff
@@ -30,7 +30,27 @@ spec:
               name:       concent-storage
             - mountPath:  /etc/nginx/conf.d/
               name:       nginx-storage-config
+            - mountPath:  /usr/local/openresty/nginx/conf/snippets/
+              name:       nginx-storage-snippets
+              readOnly:   true
```

Comment:
Why is this config file not in `/etc/nginx/` along with the rest of nginx config?

Reply:
Nginx in openresty docker is in a different location ` /usr/local/openresty/nginx/`. However  `nginx.conf` is set to add configs from `/etc/nginx/conf.d` directory. I move config to `/etc/nginx/`.